### PR TITLE
User interface - Milestone 2

### PR DESCRIPTION
**Pull request by phfdonda**
This is the Second Milestone for the Tic-Tac-Toe Game for Microverse Curriculum

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team   
#### Great Job so far, you are almost there .
### General
-  [ ] Please lets take a step back and look at the guidelines below


### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by phfdonda** [comment]
Hi, @kobiyoyo. Thank you for the review.
I don't understand what is wrong with the project. Could you please tell me? Thanks!

**Comment by phfdonda** [comment]
I think you meant the path problem, I fixed that. Was that it?

File: bin/main [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+puts 'Let the game begin!'
+
+puts ' *****************'
+
+[[" 1 | 2 | 3 "], [" 4 | 5 | 6 "], [" 7 | 8 | 9 "]].each { |x| puts x }
+
+puts 'To play, choose a number'
+
+choice = gets.chomp
+
```
Comment (majovanilla) [inline bin/main:46]: What happens if the user selects an invalid option? Remember the objective of this milestone:
Screenshot from 2020-04-05 23-02-32

File: bin/main [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+puts "It's #{player_1}'s turn"
+puts "It's #{player_2}'s turn"
+
+winner = 'Nobody'
+
```
Comment (majovanilla) [inline bin/main]: what happens if no one wins??

File: bin/main [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+[[" 1 | 2 | 3 "], [" 4 | 5 | 6 "], [" 7 | 8 | 9 "]].each { |x| puts x }
+
+puts 'To play, choose a number'
+
+choice = gets.chomp
+
+puts "It's #{player_1}'s turn"
+puts "It's #{player_2}'s turn"
+
```
Comment (majovanilla) [inline bin/main]: Think about the loop you will have to make to make the turns.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Good work team!! :star: 

You have almost completed this milestone! :muscle: 

Please take a look at the comments below. Remember the instructions posted on the image above by the previous TSE: the objective of this milestone is to have all the game flow _before_ actually coding anything.

You would be surprised of how much :timer_clock: you will save by thinking and :memo:  any project you build... designing the general outline before writing the first line of code!! This is something you should implement on all future and complex projects.

### Please make the necessary adjustments. 
Remember it can be added as comments, but you nee to think of all the flow of your game :space_invader: 

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by phfdonda** [comment]
Thank you, @majovanilla! That was an excellent review, really well explained! I tried to give a more robust explanation about the game flow in the comments inside the code, I hope that's what is expected.

**Review by whiz25 — APPROVED** [review]
## [APPROVED!] :+1: 
Merge and proceed to the next milestone.

### Martin Afani

**Pull request merged**